Cargo.toml

```diff
@@ -45,6 +44,8 @@ picky-asn1-x509 = { version = "0.3.2", optional = true }
 users = "0.10.0"
 libc = "0.2.77"
 anyhow = "1.0.32"
+# Using a fork until the JWT support is merged into the main rust-spiffe repository
+spiffe = { git = "https://github.com/hug-dev/rust-spiffe", branch = "refactor-jwt" }
```

Comment:
This means we cannot upstream to crates.io, right?

Reply:
Yes true, I forgot about that... I will try to get this upstreamed on crates.io before the next release and if not we can isolate it in a feature and not upstream that